extensions/wikia/CreateNewWiki/tasks/TaskHelper.php

```diff
@@ -58,7 +58,7 @@ public static function waitForSlaves( TaskContext $taskContext, $functionName ){
 	 * @return string[][]
 	 */
 	public static function getLoggerContext( TaskContext $taskContext, $additionalValues = [] ) {
-		$context = $taskContext->getAllProperties();
+		$context = $taskContext->getLoggerContext();
 
 		$context = array_merge( $context, $additionalValues );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$context". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)